jdbc/docker-compose-integration.yml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: dbuser
+      POSTGRES_PASSWORD: dbpass
+      POSTGRES_DB: app
+    volumes:
+      - "./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d"
+    ports:
+      - 5432
+
+  app:
+    image: jeanblanchard/java
```

Comment:
I think we should use *schibstedspain/java8-oracle* instead of this *jeanblanchard/java* image